config/strategies/github.js

```diff
@@ -23,8 +23,14 @@ module.exports = function() {
 			providerData.refreshToken = refreshToken;
 
 			// Create the user OAuth profile
+			var displayName = profile.displayName.trim();
+			var firstName =  displayName.indexOf(' ') !== -1 ? displayName.substring(0, displayName.indexOf(' ')) : displayName;
+			var lastName = displayName.indexOf(' ') !== -1 ? displayName.substring(displayName.indexOf(' ') + 1) : undefined;
```

Comment:
Since we're using indexOf more than once, why don't we cache that value in a variable?